tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -434,12 +434,15 @@ generate_qemu_options() {
 		qemu_options+=(arch:"--target-list=${arch}-softmmu")
 	fi
 
-	# aarch64 need to explictly set --enable-pie
-	if [ -z "${static}" ] && [ "${arch}" = "aarch64" ]; then
-		qemu_options+=(arch:"--enable-pie")
-		# pie is conflict with plugins build for qemu 6.1.0
-		[ "${qemu_version}" == "6.1.0" ] && qemu_options+=(arch:"--disable-plugins")
-	fi
+	# SECURITY: Create binary as a Position Independant Executable,
+	# and take advantage of ASLR, making ROP attacks much harder to perform.
+	# (https://wiki.debian.org/Hardening)
+	case "$arch" in
+	aarch64) [ -z "${static}" ] && qemu_options+=(arch:"--enable-pie") ;;
+	x86_64) [ -z "${static}" ] && qemu_options+=(arch:"--enable-pie") ;;
+	ppc64le) [ -z "${static}" ] && qemu_options+=(arch:"--enable-pie") ;;
+	s390x) [ -z "${static}" ] && qemu_options+=(arch:"--enable-pie") ;;
```

Comment:
As we're doing this for all supported arches, do we still need the switch?  If so, why?

Reply:
Seconded.  Apart from that LGTM.